src/Resources/assets/controllers/table_select_controller.js

```diff
@@ -87,13 +87,15 @@ export default class extends Controller {
     updateSelectedCount() {
         const count = this.getIds().length;
 
-        if (count === 0) {
-            this.selectedCountTarget.classList.add('hidden');
-            return;
-        }
+        if(this.hasSelectedCountTarget) {
+            if (count === 0) {
+                this.selectedCountTarget.classList.add('hidden');
```

Comment:
_Probably out of scope_: All classes should be submitted via Stimulus Classes API to the controller, especially if these are coming from third-party tools and not from us.